snapcraft/internal/remote_build/_launchpad.py

```diff
@@ -14,20 +14,20 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-import datetime
 import gzip
 import logging
 import os
 import shutil
 import time
 import urllib.request
 import urllib.error
-import urllib.parse
 
+from datetime import datetime
```

Comment:
we should discuss imports. We also follow PEP8 for this https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Fixed!